.github/pull_request_template.md

```diff
@@ -13,10 +13,10 @@ This Pull Request is linked to issue (URL): [REPLACE ME]
 
 Before submitting the PR make sure the following are checked:
 
-* [ ] This Pull Request is related to one issue.
-* [ ] Commit message has a detailed description of what changed and why.
-* [ ] Tests are added or updated.
-* [ ] CHANGELOG.md and documentation files are updated.
+* [ ] This Pull Request is related to one issue
+* [ ] Commit messages has a detailed description of what is changed
+* [ ] Tests are added or updated
+* [ ] CHANGELOG.md and documentation files are updated
 * [ ] Destination branch is correct - main or release
-* [ ] Commits will be squashed upon merging.
-      
+* [ ] All changes to `release` branch have a separate PR to backport
```

Comment:
.. to main
we will never have this checked, because backport created after merging a PR

Reply:
Should I make this a bullet point instead?

Reply:
Maybe... or configure backporting bot instead

Reply:
Good call.  I forgot that's a thing.  

Also, checkboxes can be checked after merging.  But I'll make that clearer.